### PR TITLE
2.36 updates

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

File: docs/details/glossary.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Queries Per Second. A measure of the rate of queries being monitored.
 
 Component of PMM Server that enables you to analyze MySQL query performance over periods of time.
 
-## Advisors
+## Advisor checks
```
Comment (rnovikovP) [inline docs/details/glossary.md:44]:
```suggestion
## Advisors
```

We can keep here, as the description matches too.

File: docs/get-started/advisors.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# Advisors 
-Percona Advisors provide automated insights and recommendations within Percona Monitoring and Management. These proactive insights help you uncover problems before they become larger issues (e.g., security posture, configuration, performance).
+# Working with Advisor checks
```
Comment (rnovikovP) [inline docs/get-started/advisors.md:1]:
```suggestion
# Working with Advisors
```
..as the main element is Advisor

File: docs/how-to/configure.md [hunk 3]
```diff
@@ -111,17 +111,17 @@ As well as via the *PMM Settings* page, you can also disable telemetry with the
 
 When active, PMM will automatically check for updates and put a notification in the home page *Updates* dashboard if any are available.
 
-### Advisors
+### Advisor checks
```
Comment (rnovikovP) [inline docs/how-to/configure.md:114]:
```suggestion
### Advisors
```
as this is how it's named in the menu

File: docs/how-to/configure.md [hunk 4]
```diff
@@ -14,12 +14,12 @@ On the left are the selector tabs:
     - [Data Retention](#data-retention)
     - [Telemetry](#telemetry)
     - [Check for updates](#check-for-updates)
-    - [Advisors](#advisors)
+    - [Advisor checks](#advisor-checks)
```
Comment (rnovikovP) [inline docs/how-to/configure.md:17]: probably, name Advisors here too

Comment (catalinaadam) [reply docs/how-to/configure.md:17]: The UI for Advanced Settings uses "Advisor Checks"

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**